applications/SwimmingDEMApplication/python_scripts/swimming_DEM_analysis.py

```diff
@@ -33,12 +33,12 @@ def Say(*args):
     from KratosMultiphysics.mpi import *
 
     # DEM Application MPI
-    import DEM_procedures_mpi as DEM_procedures
+    import DEM_procedures_mpi as DP
```

Comment:
this way of importing python-scripts will not be supported forever (onyl as long as the python-scripts in the apps are added to the python-path), better use the full-path

```suggestion
    import KratosMultiphysics.DEMApplication.DEM_procedures_mpi as DP
```

more details [see here](https://github.com/KratosMultiphysics/Kratos/wiki/Applications-as-python-modules)

Reply:
Also to be done all at once in a future PR. There are many instances of this both in SDEM and DEM apps.